databases/week1/Lucy_Chyzhova_HW.sql

```diff
@@ -0,0 +1,52 @@
+-- Find out how many tasks are in the task table         
+SELECT COUNT(id) AS amount_of_task  -- or (*) 
+FROM task; 
+
+-- Find out how many tasks in the task table do not have a valid due date  
+SELECT COUNT(task.id) AS task_count 
+FROM task 
+WHERE due_date is null; 
+
+-- Find all the tasks that are marked as done            
+SELECT task.title, status.name AS status_as_text
+FROM task 
+INNER JOIN status ON status.id = task.status_id
+WHERE status.name like 'done';
+
+-- Find all the tasks that are not marked as done         
+SELECT task.title, status.name AS status_as_text
+FROM task 
+INNER JOIN status ON status.id = task.status_id
+WHERE status.name NOT like 'done';
+
+-- Get all the tasks, sorted with the most recently created first*/     
+SELECT task.title, task.created 
+FROM task
+ORDER BY created DESC;
+
+-- Get the single most recently created task      
+SELECT task.title, task.created 
+FROM task
+ORDER BY created DESC
+LIMIT 0, 1;
+
+-- Get the title and due date of all tasks where the title or description contains database  -
+SELECT title, due_date FROM task WHERE title LIKE '%database%' OR description LIKE '%db%'; 
```

Comment:
I won't say that this is incorrect per se, because I can understand why you searched for the word "db" in description instead of "database" (since there is no mention of the word "database" in any of the descriptions). But just for completeness the search word should be the same for both conditions.